### PR TITLE
feat: properly update logo

### PR DESCRIPTION
**Pull request by xhayper**

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -9,7 +10,9 @@ function App() {
   return (
     <div className="App">
       <div>
-        <img src="./logo.svg" className="logo" alt="Vite logo" />
+        <a href="https://github.com/electron-vite/electron-vite-react" target="_blank">
+          <img src="/electron-vite.svg" className="logo" alt="Electron + Vite logo" />
```
Comment (caoxiemeihao) [inline src/App.tsx:14]: After Electron package, can only support `file://` protocol.
```diff
- <img src="/electron-vite.svg" />
+ <img src="./electron-vite.svg" />
```

**Review by caoxiemeihao — APPROVED** [review]
👍

**Comment by xhayper** [comment]
lmk if you want me to do the same to vue project!

**Comment by caoxiemeihao** [comment]
image
This `favicon.ico` seems a bit too small. 😅
But the `favicon.` you made before works fine.

**Comment by xhayper** [comment]
Huh, it seems to only be for macOS, i will make a fix PR

**Pull request merged**